packages/runtime/datastore/src/localChannelContext.ts

```diff
@@ -38,7 +38,7 @@ import { ChannelStorageService } from "./channelStorageService";
  */
 export abstract class LocalChannelContextBase implements IChannelContext {
     public channel: IChannel | undefined;
-    private attached = false;
+    private globallyVisible = false;
```

Comment:
can we make this a getter and internally use VisibilityState somehow?

Reply:
We could but that means adding additional state to something that doesn't need it. We should be moving to the other direction where visibility state is essentially just a boolean - you are either visible or you or not. Unfortunately, the current limitations don't allow us to do it for data stores but I think that is how it should be.